authority-portal-backend/authority-portal-api/src/main/java/de/sovity/authorityportal/api/model/catalog/CatalogDataOffer.kt

```diff
@@ -25,8 +24,11 @@ data class CatalogDataOffer(
     @field:Schema(description = "Asset Title", requiredMode = Schema.RequiredMode.REQUIRED)
     val assetTitle: String,
 
-    @field:Schema(description = "Asset Datasource Availability (there are 'LIVE' and 'ON_REQUEST' Assets)", requiredMode = Schema.RequiredMode.REQUIRED)
-    val assetDataSourceAvailability: DataSourceAvailability,
+    @field:Schema(
+        description = "Asset Datasource Availability (there are 'Available' and 'On Request' Assets)",
+        requiredMode = Schema.RequiredMode.REQUIRED
+    )
+    val assetDataSourceAvailability: String,
```

Comment:
Please use the enum type here. Build a mapper if necessary.
Or add another field in `CatalogQueryFields` since the one I suggested is kind of explicitly map to fit the automatic Filter population in the UI